cirq-core/cirq/ops/gateset.py

```diff
@@ -76,10 +124,17 @@ def __init__(
             description: Human readable description of the gate family.
             ignore_global_phase: If True, value equality is checked via
                 `cirq.equal_up_to_global_phase`.
+            tags_to_accept: If non-empty, only `Operations` containing at least one tag in this
+                sequence can be accepted. This parameter cannot contain tags in `tags_to_ignore`.
```

Comment:
Since we already mentioning in the "Raises" section that we'll raise a value error if `tags_to_accept` and `tags_to_ignore` contain common tags, we can avoid repeating it 2 times here in the docstring of `tags_to_accept` and `tags_to_ignore` 

For example:
```
tags_to_accept: If non-empty, only `cirq.Operation`s containing at least one tag in this sequence can be accepted.
tags_to_ignore: Any `cirq.Operation` containing at least one tag in this sequence is rejected. Note that this takes precedence over `tags_to_accept`, so an operation contains tags from both `tags_to_accept` and `tags_to_ignore` is rejected. 
```